iopipe/agent.py

```diff
@@ -87,7 +87,14 @@ def error(self, error):
     def __call__(self, func):
         @functools.wraps(func)
         def wrapped(event, context):
-            logger.debug("%s wrapped with IOpipe decorator" % repr(func))
+            # Skip if function is already instrumented
+            if getattr(func, "__iopipe_instrumented", False):
```

Comment:
Is a "False" value correct for this conditional?  I'd think that if we're checking for the function already being wrapped, we'd want to look for a true value here.

Reply:
Yes, we're looking for a `True` here, but the third argument to `getattr` is the default value. so this code is equivalent to:

```python
func.__iopipe_instrumented = False

if getattr(func, "__iopipe_instrumented"):
    ...
```

So by default the `False` will proceed with instrumentation.

Reply:
Great--thank you for the explanation!